java/test/jmri/jmrix/lenz/LenzXNetTurnoutCommandTest.java

```diff
@@ -0,0 +1,365 @@
+package jmri.jmrix.lenz;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+import jmri.util.JUnitUtil;
+import jmri.Turnout;
+import org.junit.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Checks Lenz clone's specific behaviour for Turnout commands.
+ * Assures that the Turnout will send an appropriate number of Accessory Request
+ * Output OFF commands, delayed properly and that it won't ping indefinitely.
+ */
```

Comment:
Make sure you include author and copyright information here.

Reply:
Isn't there a policy to assign Copyright to JMRI representative for easier legal processes ?

Reply:
We still mark the code with the author and copyright date.